app/domain/queries/find_content.rb

```diff
@@ -62,10 +60,12 @@ def array_to_hash(array)
   end
 
   def slice_editions
-    Dimensions::Edition.by_organisation_id(@organsation_id)
+    editions = Dimensions::Edition.by_organisation_id(@content_filters.organisation_id)
+    editions = editions.where('latest.document_type = ?', @content_filters.document_type) if @content_filters.document_type
```

Comment:
This may exclude editions that are latest for a given time period, but not the latest of all time. E.g. "last-month"
We should also be consistent with how we filter by `organisation_id`.